demisto_sdk/commands/common/constants.py

```diff
@@ -1633,6 +1633,7 @@ class PB_Status:
 FEATURE_BRANCHES = ["v4.5.0"]
 VERSION_REGEX = r"(\d{1,2}\.){2}\d{1,2}$"
 DOC_FILE_IMAGE_REGEX = r"[\.\./]*doc_files/[a-zA-Z0-9_-]+\.png"
+DOC_FILE_FULL_IMAGE_REGEX = r"!\[.*?\]\(((?!http).*?doc_files.*?)\)"
```

Comment:
`(?!http)` in order to catch only files in the repo and `doc_files` in order to catch only files that are under doc_files folder. RM108 should validate all images are located there - [related ticket](https://jira-dc.paloaltonetworks.com/browse/CIAC-10606).